autogluon/utils/forecasting/utils/loaders/load_pickle.py

```diff
@@ -0,0 +1,9 @@
+import pickle
+import os
+
+
+def load(path):
```

Comment:
Can you use the existing pickle-load function instead?

https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/utils/loaders/load_pkl.py#L8